### PR TITLE
add missing registration status

### PR DESCRIPTION
**Pull request by zion-splitwise**
Add `unsubscribed`

image

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -2227,8 +2227,8 @@
...
+<p>If you’re interested in integrating your commercial application with Splitwise, we strongly encourage you to contact <a href="mailto:&#x64;&#x65;&#x76;&#x65;&#108;&#x6f;&#112;&#x65;&#114;&#115;&#x40;&#115;&#x70;&#x6c;&#x69;&#116;&#x77;&#x69;&#115;&#101;&#46;&#x63;&#x6f;&#x6d;">&#x64;&#x65;&#x76;&#x65;&#108;&#x6f;&#112;&#x65;&#114;&#115;&#x40;&#115;&#x70;&#x6c;&#x69;&#116;&#x77;&#x69;&#115;&#101;&#46;&#x63;&#x6f;&#x6d;</a> so our development team can help discuss your use case, provide private APIs and Enterprise support, and offer an appropriate  commercial license for the integration. The Self-Serve API documented here may be suitable for internal prototyping and other exploratory work.</p>
```
Comment (dylanrainwater) [inline docs/index.html:2230]: These URL encodings changed. The original mailto address here was: `'developers@splitwise.com">developers@splitwise.com'`

And it changed to: `'developers@splitwise.com">developers@splitwise.com'`... Confusing. Maybe these are just two different encodings for the same thing.. for some reason?

File: docs/index.html [hunk 2]
```diff
@@ -2227,8 +2227,8 @@
...
+<p>If you are developing a non-commercial plugin application or personal project, we recommend you make use of the Self-Serve API documented here under the API Terms Of Use. Please be aware that our Self-Serve API has conservative rate and access limits, which are subject to change at any time and not well suited to commercial projects. If this is a problem for your use case, please contact us at <a href="mailto:&#x64;&#x65;&#x76;&#x65;&#108;&#111;&#x70;&#x65;&#x72;&#115;&#x40;&#115;&#x70;&#x6c;&#105;&#x74;&#119;&#x69;&#115;&#x65;&#46;&#x63;&#111;&#109;">&#x64;&#x65;&#x76;&#x65;&#108;&#111;&#x70;&#x65;&#x72;&#115;&#x40;&#115;&#x70;&#x6c;&#105;&#x74;&#119;&#x69;&#115;&#x65;&#46;&#x63;&#111;&#109;</a> to discuss your needs.</p>
```
Comment (dylanrainwater) [inline docs/index.html:2231]: I'm not even sure why these changed though

**Review by dylanrainwater — APPROVED** [review]
I'll leave it up to you if you want to merge this change

**Comment by zion-splitwise** [comment]
Scrapping this!  Slack thread [here](https://splitwise.slack.com/archives/C7W4V8GSF/p1712671558104099?thread_ts=1712670750.678949&cid=C7W4V8GSF) for context

**Pull request closed without merging**